### ISSUE TITLE
retroarch: error while loading shared libraries: libass.so.5: No such file or directory

### ISSUE THREAD

**Issue by Terramoto** [bite-your-idols/Gamestarter#87]
Hello,

I'm attempting to setup the emulator on openELEC which seems to be using libass.so.9 (read-only FS).

**Comment by bite-your-idols** [comment]
https://github.com/bite-your-idols/Gamestarter/issues/83

**Comment by amayatsky** [comment]
@bite-your-idols Hi there! I have the same issue even with the outdated v2.9. Any way to fix that? :)
Ty!

**Comment by bite-your-idols** [comment]
are you using OpenELEC?? the sfix is so easy... move to LibreELEC!!

**Comment by amayatsky** [comment]
Unfortunately, Libre was failing to properly play movies without lag on my
machine so I switched to Open. Don't know the reason, never bothered to
find out.
I've managed to run retroarch from v3.0 of the add-on by adding the missing
libraries from the Hyperion project, looks like they were removed from Open
v8 compared to v7.
You are adding libass library anyway. Maybe it's possible to throw in a
couple of other ones? :)
